src/Models/EmailLink.php

```diff
@@ -50,7 +50,11 @@ public function getDescription(): string
 
     public function getURL(): string
     {
-        return $this->Email ? sprintf('mailto:%s', $this->Email) : '';
+        $url = $this->Email ? sprintf('mailto:%s', $this->Email) : '';
+
+        $this->extend('updateGetURL', $url);
+
+        return $url;
```

Comment:
Please set this extension hook in `Link::getURL()` directly instead and name it `updateURL`. Then in these subclasses make this change:
```suggestion
        $this->beforeExtending('updateURL', function (string &$url) {
            $url = $this->Email ? sprintf('mailto:%s', $this->Email) : '';
        });
        return parent::getURL();
```